src/test/graphene_tests.cpp

```diff
@@ -100,7 +100,7 @@ BOOST_AUTO_TEST_CASE(graphene_set_decodes_multiple_sizes)
 
 BOOST_AUTO_TEST_CASE(graphene_set_finds_optimal_settings)
 {
-    const int SERIALIZATION_OVERHEAD = 10;
+    const int SERIALIZATION_OVERHEAD = 11;
     FastRandomContext insecure_rand(true);
```

Comment:
I don't really understand why you need this number at all?  Why do you need to adjust the serialized size to be something other than what it is, what's the purpose?

Reply:
Yeah you're right. I'm subtracting the same amount from every object so it won't make a difference in the grid search. I was being pedantic in wanting the estimated size to identically match the serialization size less overhead. I think I like including the serialization overhead so that I have encoded the difference between predicted and actual size somewhere (for times when I want to do some theoretical analysis of graphene performance). But I'm fine removing that adjustment from the unit test.